docs/reference/commandline/network_rm.md

```diff
@@ -10,18 +10,35 @@ parent = "smn_cli"
 
 # network rm
 
-    Usage:  docker network rm [OPTIONS] NAME | ID
+    Usage:  docker network rm [OPTIONS] NETWORK [NETWORK...]
 
-    Deletes a network
+    Deletes one or more networks
 
       --help=false       Print usage
 
-Removes a network by name or identifier. To remove a network, you must first disconnect any containers connected to it.
+Removes one or more networks by name or identifier. To remove the network, you must first disconnect any containers connected to it.
+
+To remove one network named 'my-network':
 
 ```bash
   $ docker network rm my-network
 ```
 
+You can also delete multiple networks in single 'docker network rm' command, for example:
```

Comment:
I think we can these two examples to a single one, e.g.:

```Markdown
To delete multiple networks in a single `docker network rm` command, provide multiple network names or id's. The following example deletes a network with id `3695c422697f` and a network named `my-network`:
```

@moxiegirl wdyt?